docs/cli/getting-started.md

```diff
@@ -145,12 +175,20 @@ If you want the very latest, you can also build and install tanzu CLI, and its p
 - If you additionally want to build and install CLI and plugins for all platforms, run:
 
   ```sh
-  TANZU_CLI_NO_INIT=true make build-install-cli-all
+  make build-install-cli-all
   ```
 
-The CLI currently contains a default distribution which is the default set of plugins that should be installed on
-initialization. Initialization of the distributions can be prevented by setting the env var `TANZU_CLI_NO_INIT=true`.
-Check out this [doc](../cli/plugin_implementation_guide.md#Distributions) to learn more about distributions in Tanzu CLI
+The CLI has 2 different types of plugins.
+
+  1. Standalone plugins: independent of the CLI context
+  2. Context(server) scoped plugins: scoped to one or more contexts
+
+When building the CLI locally and installing plugins with `make build-install-cli-local` or `make build-install-cli-all`, all plugins are treated as standalone plugins.
```

Comment:
does that mean there is a build time flag that determines this? if so, can we call out what that is here for clarity?